.cirrus.yml

```diff
@@ -93,6 +93,8 @@ macos12_task:
     OS: osx
     # 12 CPU cores and 24 GB of memory are available
     N: 12
+    # Temporarily use LDC until a release with a fix for issue 22942 is available (probably 2.101)
```

Comment:
I would suggest adding `FIXME:` to avoid potentially forgetting about this indefinitely and change the name of the tasks, otherwise, people think it's being compiled on DMD.

Reply:
Added the `FIXME:`. Initially renamed the job but Github still waited for the old job - even though the new job was run successfully. So I would rather keep it for now to unblock the CI.

Reply:
> Added the `FIXME:`. Initially renamed the job but Github still waited for the old job - even though the new job was run successfully. So I would rather keep it for now to unblock the CI.

This is because Github has `required` on that CI job for that branch in their rules. You need to go to `Settings`, somewhere in `Protected Branches` and deselect that job from being required.

Reply:
I don't have permissions, so I can't do it, but probably @RazvanN7 or someone else can do that.

Reply:
You're probably right. Hardly worth the effort though given that these changes will need to be reverted soon (TM)